packages/web/pages/login.tsx

```diff
@@ -1,62 +1,43 @@
 import { Flex, Image, SimpleGrid, Text } from '@metafam/ds';
-import React from 'react';
+import React, { useState } from 'react';
 
+import { PageContainer } from '../components/Container';
 import { MetaLink } from '../components/Link';
-import BackgroundImage from '../public/images/background.png';
+import { RegisterPlayer } from '../components/RegisterPlayer';
+import { WelcomePlayer } from '../components/WelcomePlayer';
+import BackgroundImage from '../public/images/login-background.png';
```

Comment:
for the future, `jpg` is better optimized for large background images than png, we could also think about webP

Reply:
Sure @krzysu  I did not know this.. I can update these images with jpg (exported as jpg from figma) in the future 
=> I tried converting the jpeg to webp, and the image size increase! So I think jpeg is the better option here.
Example sizes: 
background.png 842.9KiB
background.jpg 367.9KiB
background.webp 478.9KiB